src/main/java/org/elasticsearch/search/aggregations/metrics/stats/extended/ExtendedStatsAggregator.java

```diff
@@ -167,19 +198,25 @@ public void doClose() {
 
     public static class Factory extends ValuesSourceAggregatorFactory.LeafOnly<ValuesSource.Numeric> {
 
-        public Factory(String name, ValuesSourceConfig<ValuesSource.Numeric> valuesSourceConfig) {
+        private double sigma = 2;
```

Comment:
usually we leave these to be set in the constructor so that the Parser class takes care of setting default values. Also these variables should probably be final